app/controllers/authors_controller.rb

```diff
@@ -0,0 +1,87 @@
+class AuthorsController < ApplicationController
+  before_action :set_author,
+                only: %i(show add_to_favorite remove_from_favorite)
+  before_action :set_book_by_author, only: :show
+  before_action :load_favorite, only: %i(add_to_favorite remove_from_favorite)
```

Comment:
```suggestion
  before_action :load_favorite, only: %i(remove_from_favorite)
```
Action add k cần load